### DISCUSSION TITLE
How to handle unintended locales

### DISCUSSION THREAD

**Discussion by mengqing** [i18next/next-i18next#2301] [Q&A]
I have this nexti18next config and I use folder based config. I'm using Nextjs v14.2.5 with pages routes

```
module.exports = {
  i18n: {
    locales: [
      "en-US",
      "en-CA"
    ],
    localeDetection: false,
    defaultLocale: "en-US",
  },
  localePath: "/public/static/locales"
};
```

I would have thought that `defaultLocale` would be used as a fallback, however when I goto `www.example.com/en-GB` the page redirects to 404.

Is there a way for `defaultLocale` to be used like the name suggests?

**Comment by adrai** [comment]
defaultLocale and localeDetection is not managed by next-i18next but by Next.js itself...
https://nextjs.org/docs/pages/building-your-application/routing/internationalization

```js
    // These are all the locales you want to support in
    // your application
    locales: ['en-US', 'fr', 'nl-NL'],
    // This is the default locale you want to be used when visiting
    // a non-locale prefixed path e.g. `/hello`
    defaultLocale: 'en-US',
```
